.codex/skills/batch-merge/SKILL.md

```
@@ -0,0 +1,46 @@
+---
+name: batch-merge
+description: Merge all ready PRs in a parallel batch into develop sequentially, auto-resolving trivial CHANGELOG and documentation conflicts, pausing for human input on non-trivial ones, and running post-merge-cleanup for each successfully merged PR. Use when a parallel batch of PRs is ready for merge.
+---
+
+# Batch Merge
+
+Follow `docs/ai/development-workflow/protocols/94-batch-merge-protocol.md` exactly.
+
+1. **Auto-discovery mode** (no explicit PR numbers): run:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)